target/linux/kirkwood/patches-5.10/108-dns325.patch

```diff
@@ -0,0 +1,116 @@
+--- a/arch/arm/boot/dts/kirkwood-dnskw.dtsi
++++ b/arch/arm/boot/dts/kirkwood-dnskw.dtsi
```

Comment:
Please add:
-  alias section for status led, example:
 https://github.com/openwrt/openwrt/blob/4bbaf7db62e9addd4845d9353cdf7d7b060307f9/target/linux/kirkwood/files/arch/arm/boot/dts/kirkwood-c200-v1.dts#L16-L21
- Trigger for ata leds, example:
https://github.com/openwrt/openwrt/blob/4bbaf7db62e9addd4845d9353cdf7d7b060307f9/target/linux/kirkwood/files/arch/arm/boot/dts/kirkwood-c200-v1.dts#L93
https://github.com/openwrt/openwrt/blob/4bbaf7db62e9addd4845d9353cdf7d7b060307f9/target/linux/kirkwood/files/arch/arm/boot/dts/kirkwood-c200-v1.dts#L104
- trigger for usb, example:
https://github.com/openwrt/openwrt/blob/ec6293febc244d187e71a6e54f44920be679cde4/target/linux/kirkwood/patches-5.10/107-03-nsa325.patch#L51

Reply:
aliases: done
trigger for usb-host: done
sata: 
white LEDs indicating sat activity are not available via GPIO. And they work out of the box.
red LEDs can be used as an indication of HDD health based on SMART status. as an option